src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +11,9 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        if (findByEmail(email) != null) {
+            return findByEmail(email).getPassword().equals(password);
+        }
         return false;
```

Comment:
Simplify it using one return, findByEmail(email) - I see duplication of code, you can save it in variable